### ISSUE TITLE
Fatal Error

### ISSUE THREAD

**Issue by toxicfox777** [maromalo/Everything-Meth#2]
04:21:29 PM FATAL ERROR:  (C:\projects\payday2-superblt\src\InitiateState.cpp:320) mods/Everything Meth/lua/dialogmanager.lua:35: attempt to index global 'DialogManager' (a nil value)
stack traceback:
	mods/Everything Meth/lua/dialogmanager.lua:35: in main chunk
	[C]: in function 'dofile'
	mods/base/base.lua:158: in function 'RunHookFile'
	mods/base/base.lua:150: in function 'RunHookTable'
	mods/base/base.lua:178: in function 'OrigRequire'
	mods/[CORE] BeardLib/Core.lua:374: in function 'require'
	[string "lib/setups/gamesetup.lua"]:11: in main chunk
	[C]: in function 'require'
	mods/base/base.lua:177: in function 'OrigRequire'
	mods/[CORE] BeardLib/Core.lua:374: in function 'require'
	[string "lib/setups/networkgamesetup.lua"]:1: in main chunk
	[C]: in function 'require'
	mods/base/base.lua:177: in function 'OrigRequire'
	mods/[CORE] BeardLib/Core.lua:374: in function 'require'
	[string "lib/entry.lua"]:6: in main chunk
	[C]: in function 'require'
	mods/base/base.lua:177: in function 'OrigRequire'
	mods/[CORE] BeardLib/Core.lua:374: in function 'require'
	[string "core/lib/coreentry.lua"]:19: in main chunk

**Comment by maromalo** [comment]
When did the game crash?

**Comment by toxicfox777** [comment]
I can’t remember. I’ll play with the mod on tomorrow to see if i can get it again. But as you saw someone else had the same issue.

**Comment by lnx00** [comment]
I've seen a similar error in the log file. It doesn't crash the game for me, but should probably still be fixed:
```
02:18:22 AM FATAL ERROR:  (C:\projects\payday2-superblt\src\InitiateState.cpp:320) mods/Everything Meth/lua/dialogmanager.lua:35: attempt to index global 'DialogManager' (a nil value)
stack traceback:
	mods/Everything Meth/lua/dialogmanager.lua:35: in main chunk
	[C]: in function 'dofile'
	mods/base/base.lua:158: in function 'RunHookFile'
	mods/base/base.lua:150: in function 'RunHookTable'
	mods/base/base.lua:178: in function 'OrigRequire'
	mods/BeardLib/Core.lua:374: in function 'require'
	[string "lib/setups/gamesetup.lua"]:11: in main chunk
	[C]: in function 'require'
	mods/base/base.lua:177: in function 'OrigRequire'
	mods/BeardLib/Core.lua:374: in function 'require'
	[string "lib/setups/networkgamesetup.lua"]:1: in main chunk
	[C]: in function 'require'
	mods/base/base.lua:177: in function 'OrigRequire'
	mods/BeardLib/Core.lua:374: in function 'require'
	[string "lib/entry.lua"]:6: in main chunk
	[C]: in function 'require'
	mods/base/base.lua:177: in function 'OrigRequire'
	mods/BeardLib/Core.lua:374: in function 'require'
	[string "core/lib/coreentry.lua"]:19: in main chunk

02:18:22 AM Lua: [ERROR] [Hooks] Could not hook function 'sync_teammate_progress' (EverythingMeth_UnitNetworkHandler) 
```

**Comment by toxicfox777** [comment]
Yh, it stopped crashing but just started crashing again with the same error when I try to load into a game

**Comment by maromalo** [comment]
Fixed in v1.22
